java/AxivityAx3Epochs.java

```diff
@@ -156,6 +165,7 @@ public static void main(String[] args) {
 				} else if (funcName.equals("timeZoneOffset")) {
 					timeZoneOffset = Integer.parseInt(funcParam);
 				} else if (funcName.equals("verbose")) {
+					logger.setLevel(Level.ALL);
```

Comment:
Excuse my ignorance of using a class logger, but is there a possibility the log file can create conflicts when we do multi-CPU processing on the cluster?

Is there any easy way to toggle the logger on/off?

Reply:
Yes, this can be easily turned off by setting the level to `none`. More references here https://docs.oracle.com/javase/8/docs/api/java/util/logging/class-use/Level.html. In terms of potential conflict with the cluster processing, I am not sure as I am not familiar with how the multi-CPU processing is done. 

Have a logger for the V2 support is really helpful because we are processing lots of smaller packets. Being able to toggle different levels of logging is really useful for debugging. Maybe the best way is to remove the logger level for the `verbose` mode and set the level to None for the default mode? 

We can update this after we make sure that there is no conflict with the cluster processing?

Reply:
Yes, that sounds good. So can we go with Level.OFF as the default - and then switch logging on if we go with Verbose=True ?

Reply:
Just set the default mode to `off` and one just change the logging level by modifying the `logging.properties` file https://github.com/activityMonitoring/biobankAccelerometerAnalysis/pull/24/commits/bcfdc2d093ffd5b27a0a3ed16abee133bdec4f3a

Reply:
Many thanks. Before merging this, let's just wait unto you are happy with the changes made last night. Great work!

Reply:
Think it should be ready to go. The timestamp had some issues with the unit but has been corrected. The npy file seems reasonable. :D One can just test via: `java AxivityAx3Epochs test.gt3x npyOutput:true npyFile:test.npy`

Reply:
Many thanks @angerhang  - great work!